### PR TITLE
No more awkward hug misclicks

### PR DESCRIPTION
**Pull request by VTCobaltblood**
Changes clicking a person with an empty hand on help intent from hugs to tapping.

If you tap someone while being on fire, you also set them on fire with 1 fire stack. This sends a message to admins. You can't give them more than 1 fire stack this way since the help interaction then changes to attempting to put their fires out (which you can't do while on fire).

No other help interactions were changed - you're still going to shake people and pat the fire out of them.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
!review

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 1]
```diff
@@ -318,17 +318,29 @@
 		for(var/obj/item/organ/I in H.internal_organs)
 			I.status |= ORGAN_ADV_ROBOT
 
-/datum/species/proc/hug(var/mob/living/carbon/human/H,var/mob/living/target)
```
Comment (Sindorman) [inline code/modules/mob/living/carbon/human/species/species.dm]: I am not a fan of removing hug mechanics. Please add it as either *emote or something else. But it should still remain in game.

Comment (VTCobaltblood) [reply code/modules/mob/living/carbon/human/species/species.dm]: image

Comment (Sindorman) [reply code/modules/mob/living/carbon/human/species/species.dm]: Well it's fine if you make default help click as tap, but please keep the hug proc in and make it as emote or something else for people who actually want to hug. Plus it has a sound for it.

Comment (VTCobaltblood) [reply code/modules/mob/living/carbon/human/species/species.dm]: But people can already hug themselves via me "", and there ALSO seems to be an existing emote? Why exactly should the hugging mechanics be preserved? Removing them, surprisingly enough, does not disallow you to hug.

File: code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm [hunk 2]
```diff
@@ -110,9 +110,9 @@
 
 	return 0
 
-/datum/species/xenos/hug(var/mob/living/carbon/human/H,var/mob/living/target)
-	H.visible_message("<span class='notice'>[H] caresses [target] with its scythe-like arm.</span>", \
-					"<span class='notice'>You caress [target] with your scythe-like arm.</span>")
+/datum/species/xenos/tap(var/mob/living/carbon/human/H,var/mob/living/target)
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm]: Would argue for keeping this. It's pretty neat in its spoop.

Comment (Sindorman) [reply code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm]: Well I think we should keep both, but make tapping as default help click. And have hug as like emote or me.

Comment (VTCobaltblood) [reply code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm]: alright i'll make benos hug

Comment (VTCobaltblood) [reply code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm]: leaving the proc name like this tho but the text is a hug

**Pull request merged**